packages/astro-feather-icons/README.md

```diff
@@ -1,45 +1,81 @@
-# astro-feather-icons
+# Astro Feather Icons
 
-[GitHub](https://github.com/edazpotato/astro-feather-icons) |
-[NPM](https://npmjs.com/package/astro-feather-icons)
+Feather is a collection of simply beautiful open source icons.
+Each icon is designed on a 24x24 grid with an emphasis on simplicity, consistency, and flexibility.
 
-## Install
+```shell
+npm install astro-feather-icons
 
-```bash
-npm i astro-feather-icons --save
 pnpm i astro-feather-icons
+
+yarn add astro-feather-icons
 ```
 
+[GitHub](https://github.com/edazpotato/astro-feather-icons) |
+[NPM](https://npmjs.com/package/astro-feather-icons)
+
+
+
 ## Usage
 
+All of the icons are available from a single import, and only the icons that are used will be added to the page.
+
 ```astro
 ---
-import { AirplayIcon, AtSignIcon, ... } from 'astro-feather-icons'
+import * as Icon from 'astro-feature-icons'
 ---
+<Icon.AirPlay />
+<Icon.AtSign />
+<Icon.Zap size="60" fill="gold" />
+```
 
-<AirplayIcon size="24" />
-<AtSignIcon size="1.5x" />
+The icons are also available as individual imports.
+
+```astro
+---
+import AirPlayIcon from 'astro-feature-icons/AirPlay'
+import AtSignIcon from 'astro-feature-icons/AtSign'
+import ZapIcon from 'astro-feature-icons/Zap'
```

Comment:
```suggestion
import AirPlayIcon from 'astro-feather-icons/AirPlay'
import AtSignIcon from 'astro-feather-icons/AtSign'
import ZapIcon from 'astro-feather-icons/Zap'
```

Reply:
Thank you! Fixed in https://github.com/edazpotato/astro-feather-icons/pull/1/commits/32d4d5920ec4625a79b1702021215c301f03d060